src/main/java/org/spectrumauctions/sats/core/bidfile/JsonExporter.java

```diff
@@ -68,7 +68,7 @@ private JsonElement singleBidderXOR(XORLanguage<? extends Good> lang, int number
                 licenses.add(license.getId());
             }
             bid.add("licenses", licenses);
-            bid.addProperty("value", roundedValue(xorValue.value().doubleValue()));
+            bid.addProperty("value", xorValue.value().setScale(4, BigDecimal.ROUND_HALF_UP).toString());
```

Comment:
Please define the 4 as a constant somewhere in case we want to change that one day... it's used all over the place ;-)